scripts/publish.sh

```diff
@@ -124,6 +124,14 @@ hyderabad_region="ap-south-2"
 
 hyderabad_account_id="378905956269"
 
+tel_aviv_region="il-central-1"
```

Comment:
I wonder why its `il-central` ? No other `il` regions. And how did they pick `il`. 🤔

Reply:
israel!